src/Sarif.Driver/Sdk/PluginDriverCommand.cs

```diff
@@ -95,5 +179,61 @@ protected virtual void ProcessBaseline(IAnalysisContext context, T driverOptions
                 };
             }
         }
+
+        protected virtual void PostLogFile(string postUri, string outputFilePath, IFileSystem fileSystem)
+        {
+            PostLogFile(postUri, outputFilePath, fileSystem, new HttpClient())
+                .GetAwaiter()
+                .GetResult();
+        }
+
+        public static async Task PostLogFile(string postUri, string outputFilePath, IFileSystem fileSystem, HttpClient httpClient)
```

Comment:
>public static [](http://example.com/codeflow?start=8&length=13)

changed the method to public, so people who uses the library can call this method.

Now, I'm wonder if we should make this an extension of the SarifLog itself.

```
Task SarifLog.Post(string postUri, string filePath);

Task SarifLog.Post(string postUri, Stream stream)
```

With that, we could keep the PostLogFile internal

@cfaucon @michaelcfanning